src/main/java/smartthings/konsumer/KafkaListener.java

```diff
@@ -34,6 +40,7 @@ public KafkaListener(ListenerConfig config) {
 		consumer = Consumer.createJavaConsumerConnector(config.getConsumerConfig());
 		topic = config.getTopic();
 		streamProcessor = new StreamProcessorFactory(config).getProcessor();
+		circuitBreaker = new SimpleCircuitBreaker();
```

Comment:
Lets leave that for later.  I think the implementation we have here for the circuit breaker will take people pretty far and we can always make a builder setup or whatever later if we deem necessary.